### PR TITLE
Handle detection v2

### PR DESCRIPTION
**Pull request by AndreasMadsen**
~~**work in progress**~~

The previous implementation of the sign test in `analysis/analyse-handles.js` was just simply wrong. This sign test implementation is much more mathematically correct and does detect issues correctly on all the data I have available.

However, there are still theoretical issues with detecting increasing signals. ~~I'm currently looking into non-parametric symmetry tests, in order to detect these correctly. These issues are revealed in the out-commented tests.~~

This also adds a second test. This test works directly on the handles data and is thus able to detect increasing signals. It is a non-parametric skewness test, and thus depends on the handling graph eventually flattening out. This is a theoretical constraint, but I doubt it will cause any issues. The advantage is that a screwless test is much more robust to noise and "non-linear signals" compared to more conventional "increasing" detection methods, such as linear regression.

### CODE REVIEW

**Comment by mcollina** [comment]
CI seems to be failing

**Comment by AndreasMadsen** [comment]
@mcollina there is some function on Windows that is not being tested. @goto-bus-stop says it is a known issue that he has been meaning to look into.

**Comment by mcollina** [comment]
I would just flag that as ignored by code coverage and land this.

**Comment by AndreasMadsen** [comment]
> I would just flag that as ignored by code coverage and land this.

We don't know what line it is. So it is something that needs to be debugged on a Windows computer.

File: index.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class ClinicDoctor extends events.EventEmitter {
 
     let NODE_PATH = path.join(__dirname, 'injects')
     // use NODE_PATH to work around issues with spaces in inject path
+    /* istanbul ignore if: spaces in path dependes on install location */
```
Comment (goto-bus-stop) [inline index.js]: https://github.com/nearform/node-clinic-doctor/pull/246 added a test for this `if`, so this can be un-ignored

Comment (AndreasMadsen) [reply index.js]: Thanks

**Comment by AndreasMadsen** [comment]
Windows coverage issue will be fixed in https://github.com/nearform/node-clinic-doctor/pull/248

**Pull request merged**